portability-worker/src/main/java/org/dataportabilityproject/worker/WorkerExtensionContext.java

```diff
@@ -38,11 +40,22 @@
           .put(JsonFactory.class, new JacksonFactory())
           .build();
 
+  private final TypeManager typeManager;
+
+  public WorkerExtensionContext() {
+    this.typeManager = new TypeManagerImpl();
```

Comment:
Should we add a todo to use create an initialize a TypeManager with all the registered types?

Reply:
OK let me add that